hazelcast/src/main/java/com/hazelcast/monitor/TimedMemberState.java

```diff
@@ -190,12 +190,9 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        StringBuilder sb = new StringBuilder("TimedMemberState{\n");
-        sb.append("\t");
+        StringBuilder sb = new StringBuilder("TimedMemberState{\n\t");
```

Comment:
if you are changing the `toString()` method then I'd suggest to make it platform neutral.